web/js/gerbera.app.js

```diff
@@ -101,7 +101,7 @@ GERBERA.App = (function () {
   var loadConfig = function (response) {
     var deferred;
     if (response.success) {
-      GERBERA.App.serverConfig = response.config
+      GERBERA.App.serverConfig = $.extend({}, response.config)
```

Comment:
Directly setting the value without extending was causing bad assertions for later tests when things ran too fast!! :leopard: